cdap-proto/src/main/java/io/cdap/cdap/proto/operationrun/OperationRun.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.proto.operationrun;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+/**
+ * This class records information for a particular operation run.
+ */
+public class OperationRun {
```

Comment:
`OperationRun` is a bit redundant, I think we should just call it `Operation`

Reply:
As mentioned above we already have a class named `Operation` used in lineage. I have used `OperationRun` here to differentiate. Happy to accept any suggestion for a better name